tools/GATK-VariantRecalibrator-Indels.cwl

```diff
@@ -17,13 +17,12 @@ doc: |
             -recalFile    $tmpDir/out.recal \
             -tranchesFile $tmpDir/out.tranches \
             -rscriptFile  $tmpDir/out.R \
-            -nt 4 \
```

Comment:
From a personal experience, I recommend using BQSR 2 steps with `-nct` and vqsr with `-nt`, I won't dive into the details that you guys discussed extensively above. However, BQSR crashes in many cases with `-nt`, and vqsr –which can be painfully slow with high-coverage data– performs very well with `-nt`.